carsus/io/cmfgen/base.py

```diff
@@ -711,7 +736,7 @@ def _get_levels_lines(self, data):
             symbol = convert_atomic_number2symbol(ion[0])
             logger.info(f'Loading atomic data for {symbol} {ion[1]}.')
 
-            lvl = reader['levels']
+            lvl = reader['levels'].copy()
```

Comment:
Why are we performing a copy() operation here?

Reply:
I did this to make sure that operations which are done right after(on the copy) do not change the original dataframe. I however reverted the change thinking this probably is not needed now.